adam_preprocessing/shape_stroke_extraction.py

```diff
@@ -38,6 +38,9 @@
 from utils import label_from_object_language_tuple
 
 
+GAMMA = 0.05
```

Comment:
I'd like to rename this something more informative, say `TOUCHING_REL_DISTANCE_THRESHOLD`.